src/packages/README.md

```diff
@@ -30,33 +103,74 @@ the [CheriBSD ports issue tracker](https://github.com/CTSRD-CHERI/cheribsd-ports
 
 **CheriABI packages** are compiled using pure-capability CHERI C/C++, and
 employ fine-grained C/C++ memory protection.
```

Comment:
```suggestion
employ fine-grained C/C++ memory protection. The primary function of these packages is to provide necessary dependencies for efforts to port software to CheriABI and to support CHERI demonstrations.
```

Reply:
I applied your suggestion but in the second paragraph.